src/devices/QwiicButton/QwiicButton.InterruptConfiguration.cs

```diff
@@ -0,0 +1,92 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// When called, the interrupt will be configured to trigger when the button is pressed.
+        /// If <see cref="EnableClickedInterrupt"/> has also been called,
+        /// then the interrupt will trigger on either a push or a click.
+        /// </summary>
+        public void EnablePressedInterrupt()
+        {
+            var interrupt =
+                new InterruptConfigBitField(_registerAccess.ReadSingleRegister(Register.InterruptConfig))
+                {
+                    PressedEnable = true
+                };
+            _registerAccess.WriteSingleRegister(Register.InterruptConfig, interrupt.InterruptConfigValue);
```

Comment:
I assume this mean UInt8 (byte) rather than single which means `float`? If so please change name

Reply:
Fixed